BigBatch/Models/ColorPalette.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ColorPalette.swift
+//  BigBatch
+//
+//  Created by James on 1/20/23.
+//
+
+import UIKit
+
+struct ColorPalette {
+    
+    let red = UIColor(red: 219/255, green:7/255, blue: 61/255, alpha: 1.0)
+    let green = UIColor(red: 0.3, green: 0.8, blue: 0.6, alpha: 1.0)
+    let teal = UIColor(red: 142/255, green: 199/255, blue: 210/255, alpha: 1.0)
```

Comment:
Lets keep these uniform, so the green would be UIColor(red: 76.5/255, green: 204/255, blue: 153/255, alpha: 1.0)